skdownscale/pointwise_models/gard.py

```diff
@@ -232,14 +274,60 @@ def predict(self, X):
             # for mean/weight cases, this fills nans when all analogs
             # were below thresh
             predicted = np.nan_to_num(predicted, nan=0.0)
-
-        if self.stats:
-            # calculate the standard deviation of the anlogs
-            if self.thresh is None:
-                self.stats_['error'] = analogs.std(axis=1)
-            else:
-                self.stats_['error'] = analogs.where(analog_mask).std(axis=1)
-                # calculate the probability of precip
-                self.stats_['pop'] = np.where(analog_mask, 1, 0).mean(axis=1)
+            self.prediction_error_ = masked_analogs.std(axis=1)
+            self.exceedance_prob_ = np.where(analog_mask, 1, 0).mean(axis=1)
+        else:
+            self.prediction_error_ = analogs.std(axis=1)
+            self.exceedance_prob_ = np.ones(len(X))
 
         return predicted
+
+
+class PureRegression(RegressorMixin, BaseEstimator):
+    _fit_attributes = ['stats_', 'logistic_model_', 'linear_model_']
```

Comment:
should we add `prediction_error_` and `exceedance_prob_` here?

Reply:
gp, added for the other two model classes as well